raiden/transfer/channel.py

```diff
@@ -819,6 +841,10 @@ def get_balance(
     )
 
 
+def lock_expiration_threshold(lock):
```

Comment:
Doesn't look like this is being used anywhere.

Reply:
ah yes, this is a left over, will remove it